app/src/main/java/app/passwordstore/ui/autofill/AutofillDecryptActivity.kt

```diff
@@ -218,9 +216,15 @@ class AutofillDecryptActivity : BasePGPActivity() {
           }
           .onSuccess { result ->
             return runCatching {
-                if (features.isEnabled(EnablePGPPassphraseCache)) {
-                  passphraseCache.cachePassphrase(this, identifiers.first(), password)
-                }
+                runCatching {
+                    if (features.isEnabled(EnablePGPPassphraseCache)) {
+                      passphraseCache.cachePassphrase(this, identifiers.first(), password)
+                      settings.edit {
+                        putBoolean(PreferenceKeys.CLEAR_PASSPHRASE_CACHE, clearCache)
+                      }
+                    }
+                  }
+                  .onFailure { e -> e.printStackTrace() }
```

Comment:
```suggestion
                  .onFailure { e -> logcat { e.asLog() } }
```

You'll have to import `logcat.asLog`.

Reply:
Thanks, I edited the line as suggested and did the same in the other file.